build/build.xml

```diff
@@ -254,13 +258,17 @@
   </macrodef>
 
   <!-- copy hardware folder -->
-  <target name="assemble-hardware" unless="light_bundle">
+  <target name="assemble-hardware" depends="assemble-hardware-full, assemble-hardware-light" />
+  <target name="assemble-hardware-full" unless="light_bundle">
     <copy todir="${target.path}/hardware">
       <fileset dir="../hardware">
         <exclude name="arduino/sam/**"/>
       </fileset>
     </copy>
   </target>
+  <target name="assemble-hardware-light" if="light_bundle">
+    <copy file="../hardware/package_index_bundled.json" todir="${target.path}/hardware" />
```

Comment:
`package_index_bundled.json` is, basically, a core-package json that describe what's bundled inside the IDE `hardware` folder. The IDE merges the content of `package_index_bundled.json` with the general `package_index.json` that is downloaded from arduino.cc. The bundled index has priority over the downloaded index.

If you want to make a real no-bundle (or light) version of the IDE, the correct thing to do is to create an empty (== with zero platforms defined) `package_index_bundled.json`.